### PR TITLE
alex: add pt_BR translation

### PR DESCRIPTION
**Pull request by juliafmorgado**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
https://github.com/tldr-pages/tldr/blob/main/pages/common/alex.md

### CODE REVIEW

File: pages.pt_BR/common/alex.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# alex
+
+> Uma ferramenta que captura escrita insensível e sem consideraçāo.
+> Ajuda a encontrar no texto, frases favorecedoras de gênero, polarizantes, relacionadas à raça, insensíveis à religiao e outras frases desiguais.
+> Mais informações: <https://github.com/get-alex/alex>.
+
+- Analisa o texto do stdin:
+
+`echo {{His network looks good}} | alex --stdin`
```
Comment (navarroaxel) [inline pages.pt_BR/common/alex.md:9]: Please translate the placeholders for get this easier for non-English speakers.

Comment (juliafmorgado) [reply pages.pt_BR/common/alex.md]: Done :)

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

File: pages.pt_BR/common/alex.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+`echo {{A rede dele parece boa}} | alex --stdin`
+
+- Analisa todos arquivos no diretório atual:
+
+`alex`
+
+- Analisa um arquivo específico:
+
+`alex {{textfile.md}}`
```
Comment (navarroaxel) [inline pages.pt_BR/common/alex.md:17]: you can translate this too.
```suggestion
`alex {{archivo_do_texto.md}}`
```

**Review by blueskyson — APPROVED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages.pt_BR/common/alex.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+`echo {{A rede dele parece boa}} | alex --stdin`
+
+- Analisa todos arquivos no diretório atual:
+
+`alex`
+
+- Analisa um arquivo específico:
+
+`alex {{archivo_do_texto.md}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/alex.md:17]: It's better to use `{{path/to/file.txt}}` in the original page and here (translated).

File: pages.pt_BR/common/alex.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+- Analisa todos arquivos em Markdown exceto `example.md`:
+
+`alex *.md !{{example.md}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/alex.md:21]: The same goes here. ;)

**Pull request merged**